shell/store/features.js

```diff
@@ -27,6 +27,7 @@ export const mapFeature = function(name) {
 export const MULTI_CLUSTER = create('multi-cluster-management', true);
 export const LEGACY = create('legacy', false);
 export const RKE2 = create('rke2', true);
+export const RKE1_UI = create('legacy', true);
```

Comment:
Wrong name

Reply:
This is to test, I will change it once I finish creating a test for this change.